chatmeleon-server/routes/message-route.ts

```diff
@@ -0,0 +1,11 @@
+import messageController from "../controllers/message-controller.js";
+import { Router } from 'express';
+
+export const messageRoute = Router();
+
+messageRoute.get('/messages/:id', messageController.getMessageById);
```

Comment:
There should be some separation lines between different methods GET, POST, PUT, PATCH, and DELETE for clarity